contribs/gnofaucet/github.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"math"
+	"os"
+	"time"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/redis/go-redis/v9"
+)
+
+type githubCfg struct {
+	rootCfg           *serveCfg
+	ghClientID        string
+	maxClaimableLimit int64
+	cooldownPeriod    time.Duration
+}
+
+func (c *githubCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.StringVar(
+		&c.ghClientID,
+		"github-client-id",
+		"",
+		"github client id for oauth authentication",
+	)
+
+	fs.DurationVar(
+		&c.cooldownPeriod,
+		"cooldown-period",
+		24*time.Hour,
+		"minimum required time between consecutive faucet claims by the same user",
+	)
+
+	fs.Int64Var(
+		&c.maxClaimableLimit,
+		"max-claimable-limit",
+		math.MaxInt64,
```

Comment:
This should be 0 by default (no limit)

Reply:
Changed to 0 = no limit :)
82c3600f317e8a29df77f24290a24298d7332314